src/20241213_AaveV3Ethereum_TokenLogicFinancialServiceProvider/TokenLogicFinancialServiceProvider.md

```diff
@@ -0,0 +1,111 @@
+
+**Liquidity**
+
+We will continue to spearhead the operations of the Aave Liquidity Committee (ALC), with a primary focus on managing liquidity and driving integrations across diverse protocols.
+
+Our key responsibilities include:
+
+- Strategically adjusting liquidity incentives to ensure robust market depth;
+- Designing and implementing direct liquidity incentive programs;
+- Leading initiatives to enhance GHO’s utility and adoption;
+- Optimizing the deployment of the DAO’s strategic voting assets;
+- Supporting future GHO facilitator programs, and Protocol-Owned Liquidity (POL) initiatives;
+- Facilitating GHO integration into centralized exchanges, fund management platforms, and the broader DeFi ecosystem; and,
+- Support teams seeking to attain a line of credit within the upcoming Emergence instance.
+
+**Parameter Configuration**
+
+TokenLogic will continue to provide detailed insights into GHO's performance and provide parameter recommendations that prioritise price stability whilst balancing the DAO's growth ambitions.
+
+Strategic focus will be placed on managing the borrow rate and peg relationship to maintain the resilience of the peg by retaining funds within the GSMs, whilst promoting the ongoing use of GHO within evolving market dynamics.
+
+The Aave Portal will provide insights into the impact of the parameter adjustments to curate an environment that promotes sustainable growth, adoption and aligns with Aave's broader financial goals.
+
+The TokenLogic team will collaborate closely with other service providers to create balanced, effective solutions that benefit the protocol as a whole.
+
+### Aave Protocol
+
+The evolution of the Aave protocol, including updates to Aave V4, the deployment of Umbrella, and the launch of new Aave markets, will bring transformative changes to the protocol’s operations. Our proposal focuses on providing the financial planning and management essential for navigating these developments and ensuring smooth integration within the broader ecosystem.
+
+In addition, we will support the adaptation of Aave’s tokenomics to meet the requirements of Umbrella and the introduction of new staking pools. Through proactive collaboration, strategic guidance, and consultation, we are committed to ensuring these updates are implemented seamlessly and positioned for long-term success.
+
+## Terms
+
+- 12-month engagement, December 15th 2024 to December 15th 2025;
+- $1M, streamed linearly throughout the engagement;
+- $1,000,000 is to be withdrawn in aEthLidoGHO (Aave V3 Lido GHO)
+
+## Specification
+
+To support funding the streams, the following is to be performed:
+
+- 0.75M GHO from the Collector deposited into the Lido Prime instance;
+- 0.75M aEthUSDC withdrawn from Aave V3 Mainnet and swapped for GHO; and,
+- GHO from above swap is to be sent to Collector (Treasury).
+
+Create the following stream allowing TokenLogic to withdraw aEthLidoGHO from the Prime instance.
+
+TokenLogic
```

Comment:
it's markdown, so these lines will be rendered as a single line
better use a list or add an empty line between